nixos/modules/virtualisation/podman/default.nix

```diff
@@ -5,12 +5,15 @@ let
 
   inherit (lib) mkOption types;
 
-  podmanPackage = (pkgs.podman.override {
+  podmanPackage = pkgs.podman.override {
     extraPackages = cfg.extraPackages
       # setuid shadow
       ++ [ "/run/wrappers" ]
       ++ lib.optional (config.boot.supportedFilesystems.zfs or false) config.boot.zfs.package;
-  });
+    extraRuntimes = lib.optionals (config.virtualisation.containers.containersConf.settings.network.default_rootless_network_cmd or "" == "slirp4netns") (with pkgs; [
+      slirp4netns
```

Comment:
Will this replace `runc`?

Reply:
I wouldn't expect it to as runc is:
`runc is a CLI tool for spawning and running containers on Linux according to the OCI specification.`

and slirp4netns/passt different user-mode networking for unprivileged network namespaces. I don't see a direct connection between this change and `runc`?

Reply:
extraRuntimes is runc by default, we are overriding it with only slirp4netns

Reply:
Thanks for pointing it out - I haven't expected extra to have a default but being empty by default 🙏🏻